Core/Src/main.c

```diff
@@ -86,15 +90,20 @@ int main(void)
 
   /* Initialize all configured peripherals */
   MX_GPIO_Init();
-  MX_USART2_UART_Init();
+  MX_SPI1_Init();
+  MX_TIM1_Init();
   /* USER CODE BEGIN 2 */
-
+  __HAL_TIM_SET_COMPARE(&htim1, TIM_CHANNEL_1, 0);
```

Comment:
set this to the minimum valid pwm input (5% duty cycle). reason for doing this is so that if code faults after starting the pwm, the servo doesn't break trying to move to an unreachable position.